docs/user-guide/configure-zowe-runtime.md

```diff
@@ -481,106 +501,163 @@ The manual installation consists of the following steps.
-        RECKEY profile-prefix ADD(profile-suffix uid(UID string for IZUSVR) SERVICE(READ) ALLOW)   ... [repeat for userids IKED, NSSD, and Policy Agent]
+        ```
+        ```
+        RECKEY profile-prefix ADD(profile-suffix uid(UID string for IZUSVR) SERVICE(READ) ALLOW)
+        ```
+        (repeat for userids IKED, NSSD, and Policy Agent)
+        
+        ```
         F ACF2,REBUILD(CSF)
         ```
-        - If you use CA Top Secret, issue the following commands. Note that `profile-prefix` and `profile-suffix` are user defined.
+        - If you use CA Top Secret, issue the following command (note that `profile-prefix` and `profile-suffix` are user defined):
+        ```
+        TSS ADDTO(owner-acid) RESCLASS(CSFSERV)                              
         ```
-        TSS ADDTO(owner-acid) RESCLASS(CSFSERV)                                                      
+        ```                  
         TSS ADD(owner-acid) CSFSERV(profile-prefix.)
+        ```
+        ```
         TSS PERMIT(tcpip-stackname) CSFSERV(profile-prefix.profile-suffix) ACCESS(READ)
-        TSS PERMIT(user-acid) CSFSERV(profile-prefix.profile-suffix) ACCESS(READ)                  ... [repeat for user-acids IKED, NSSD, and Policy Agent]
         ```
+        ```
+        TSS PERMIT(user-acid) CSFSERV(profile-prefix.profile-suffix) ACCESS(READ)
+        ```
+        (repeat for user-acids IKED, NSSD, and Policy Agent)
+    
+    **Notes**
     - The user under which zssServer runs will need READ access to CSFRNGL in the CSFSERV class.
     - Determine whether you want SAF authorization checks against CSFSERV and set `CSF.CSFSERV.AUTH.CSFRNG.DISABLE` accordingly.
     - Refer to the [z/OS 2.3.0 z/OS Cryptographic Services ICSF System Programmer's Guide: Installation, initialization, and customization](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.3.0/com.ibm.zos.v2r3.csfb200/iandi.htm).
     - CCA and/or PKCS #11 coprocessor for random number generation.
     - Enable FACILITY IRR.PROGRAM.SIGNATURE.VERIFICATION and RDEFINE CSFINPV2 if required.
 
-1. Security environment switching
+1. Configure security environment switching:
 
-    The node zssServer running under USS needs the ability to change the security environment of its process in order to associate itself with the security context of the logged in user when responding to API requests, also known as impersonation.  To switch the security environment, the user ID asscoaited with the ZOWESVR started task IZUSVR must have UPDATE access to the BPX.SERVER and BPX.DAEMON FACILITY classes.
+    When responding to API requests, the node zssServer running under USS must be able to change the security environment of its process to associate itself with the security context of the logged in user. This is called impersonation. To enable impersonation, you must grant the user ID associated with the zssServer started task UPDATE access to the BPX.SERVER and BPX.DAEMON FACILITY classes.
```

Comment:
Should we keep `ZOWESVR` as the started task under which `zssServer` binary runs? There is no `zssServer started task`.
Maybe this is better:
```
  When responding to API requests, the node zssServer running under USS must be able to change the security environment of its process to associate itself with the security context of the logged in user. This is called impersonation. To enable impersonation, you must grant the user ID associated with the zssServer started task (in our case ZOWESVR) UPDATE access to the BPX.SERVER and BPX.DAEMON FACILITY classes.
```